### PR TITLE
website: Add command 'restart' with docs & tests

### PR DESCRIPTION
**Pull request by ad-m**

### CODE REVIEW

File: bin/website/tests.js [hunk 1]
```diff
@@ -352,3 +372,20 @@ ava.serial('website snapshot receive to zfs', async t => {
...
+
+ava.serial('website restart nodejs app', async t => {
+    const password = await tests.getToken();
+    const image = 'h1cr.io/website/node:12';
+    const website = await tests.run(`website create --name ${tests.getName(t.title)} --type website-dedicated --image '${image}' --password ${password}`);
+    const content = images[image].code;
+    await mkDirWebsite(website, { password}, 'app');
+    await putFileWebsite(website, { password }, 'app/index.js', content);
+    await tests.run(`website restart --website ${website.id}`);
+    await tests.delay(5 * 1000);
```
Comment (fredericosilva) [inline bin/website/tests.js]: can we extract and name those time constants?

File: bin/website/tests.js [hunk 2]
```diff
@@ -214,14 +221,28 @@ ava.serial('website stop & start', async t => {
 
 const languages = {
     php: '<?php error_reporting(E_ALL); file_put_contents("/data/public/test.txt", "TOKEN"); ?>',
+    node: `const rand = Math.random();
+    require('http').createServer(
+        (req, res) => res.end((25).toString())
+    ).listen(require('process').env.PORT);`,
```
Comment (fredericosilva) [inline bin/website/tests.js]: is there a need to use require here?

File: bin/website/tests.js [hunk 3]
```diff
@@ -214,14 +221,28 @@ ava.serial('website stop & start', async t => {
 
 const languages = {
     php: '<?php error_reporting(E_ALL); file_put_contents("/data/public/test.txt", "TOKEN"); ?>',
+    node: `const rand = Math.random();
+    require('http').createServer(
+        (req, res) => res.end((25).toString())
```
Comment (fredericosilva) [inline bin/website/tests.js]: why 25?

File: bin/website/tests.js (hunk 1 above)
Comment (ad-m) [reply bin/website/tests.js]: Yes, of course, we can.

File: bin/website/tests.js (hunk 3 above)
Comment (ad-m) [reply bin/website/tests.js]: Typo.

File: bin/website/tests.js (hunk 2 above)
Comment (ad-m) [reply bin/website/tests.js]: No need, fixed.

File: bin/website/tests.js [hunk 4]
```diff
@@ -214,14 +222,42 @@ ava.serial('website stop & start', async t => {
...
     'h1cr.io/website/php-apache:7.2': {
         code: languages.php,
     },
     'h1cr.io/website/php-apache:5.6': {
         code: languages.php,
     },
+    'h1cr.io/website/node:10': {
+        code: languages.node,
+    },
+    'h1cr.io/website/node:11': {
```
Comment (fredericosilva) [inline bin/website/tests.js]: node:11 can be remove since we don't want to support it

**Pull request merged**